airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/source.py

```diff
@@ -128,7 +126,7 @@ def get_nested_streams(self, config: Mapping[str, Any]) -> List[Stream]:
             PostComments(**args),
             PostCommentVotes(**args),
             PostVotes(**args),
-            tickets,
+            Tickets(**args),
```

Comment:
The `tickets` wasn't re-used so I just move the instantiation here like the other streams